backend/inspirehep/submissions/tasks.py

```diff
@@ -11,7 +11,7 @@
 from inspirehep.rt.tickets import CreateTicketException, create_ticket_with_template
 
 
-@shared_task(ignore_result=False, max_retries=5)
+@shared_task(ignore_result=True, max_retries=5)
```

Comment:
I thought ticket creation was important (something to wait for) for some things like job submissions. For example, if there is no ticket nobody would check and approve it [pending -> open].

Reply:
we will get an exception on Sentry, which is currently the case anyhow no? The user is just getting an error with no information, right?

Reply:
ping @harunurhan